### PR TITLE
Nav2 path planning with waywiser

### LINKED ISSUE #35: Nav2 path planning with waywiser

### PR DESCRIPTION
**Pull request by r2avula**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

  Feature

* **What is the current behavior?** (You can also link to an open issue here)

  No dynamic path planning

* **What is the new behavior (if this is a feature change)?**

  Dynamic path planning to a fixed goal; through multiple goals; and waypoint following through multiple goals is possible. 

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

  No

* **Other information**:

Gazebo simulation with nav2: https://github.com/RISE-Dependable-Transport-Systems/WayWiseR/assets/58977950/47caeb36-a692-4ee8-97af-99623b53015c
Requires commits from https://github.com/RISE-Dependable-Transport-Systems/WayWiseR/pull/34 to test this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,15 +84,16 @@ The current state presents a foundation that our research projects [AGRARSENSE](
 WayWiseR is divided into several ROS2 packages. Make sure to have a look into the respecitve package.xml files.
 
 - **waywiser**: A meta package that depends on all packages below to be able to refer to WayWiseR as a whole.
-- **waywiser_twist_safety**: Contains configuration and launch files to manage twist commands to vehicle from different sources. It also includes a composable node for emergency stop monitoring.
 - **waywiser_description**: Contains vehicle descriptions in the form of [xacro](https://docs.ros.org/en/humble/Tutorials/Intermediate/URDF/Using-Xacro-to-Clean-Up-a-URDF-File.html) files. Currently a single vehicle is available that corresponds to a [Traxxas](https://traxxas.com/) Slash incl. camera, depth camera, LiDAR and IMU.
 - **waywiser_gazebo**: Everything related to simulation using [Gazebo](https://gazebosim.org).
 - **waywiser_hwbringup**: Configuration and launch files to get real (not simulated) vehicles running.
+- **waywiser_nav2**: Package that provides dynamic path planning using Nav2.
```
Comment (marvind) [inline README.md]:
```suggestion
- **waywiser_nav2**: Package that provides dynamic path planning using [Nav2](https://navigation.ros.org/).
```

File: waywiser_nav2/launch/nav2_bringup_all.launch.py [hunk 2]
```diff
@@ -0,0 +1,341 @@
+# Copyright (c) 2018 Intel Corporation
```
Comment (marvind) [inline waywiser_nav2/launch/nav2_bringup_all.launch.py:1]: Does this actually come from intel or was this pasted by accident?

File: waywiser_node/src/waywise_rover.cpp [hunk 3]
```diff
@@ -340,6 +359,11 @@ class WayWiseRover : public QObject, public rclcpp::Node
 
   std::string waywise_control_tower_address_;
 
+  float odom_translation_gain_;
```
Comment (marvind) [inline waywiser_node/src/waywise_rover.cpp]: This variable is not used anywhere, is it?

**Review by marvind — APPROVED** [review]
Works like a charm! Few minor comments. Great job! 🥳

File: waywiser_nav2/launch/nav2_bringup_all.launch.py (hunk 2 above)
Comment (r2avula) [reply waywiser_nav2/launch/nav2_bringup_all.launch.py:1]: It's coming from the nav2 bringup package source file: https://github.com/ros-planning/navigation2/blob/1ff1303537dfaceaa953faff7530febace585f83/nav2_bringup/launch/bringup_launch.py#L1

File: waywiser_node/src/waywise_rover.cpp (hunk 3 above)
Comment (r2avula) [reply waywiser_node/src/waywise_rover.cpp]: That's true! Good that you found this. I removed it in https://github.com/RISE-Dependable-Transport-Systems/WayWiseR/pull/37/commits/133dd3daa5e0fe63ec3e5e1c9b6ddbff887969f3

File: README.md (hunk 1 above)
Comment (r2avula) [reply README.md]: Updated it in https://github.com/RISE-Dependable-Transport-Systems/WayWiseR/pull/37/commits/1bd47555f0c87186f0f8212eeac61ee1e2ffe19a

**Pull request merged**